site/_posts/2020-07-15-octant-0.14-release.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Octant 0.14 - Now with Resource Creation"
+excerpt: Octant now supports the ability to create resources for 0.16+ Kubernetes clusters by taking advantage of server side apply.
+image: /img/posts/2020/15/create-resource.png
+author_name: Wayne Witzel III
+author_avatar: /img/contributors/wayne-witzel-iii.jpg
+categories: ['kubernetes']
+tags: ['Wayne Witzel III', 'release']
+---
+# Octant 0.14 - Create resources, port forward services, and more.
+
+Lots of new features in the [latest release of Octant](https://github.com/vmware-tanzu/octant/releases/tag/v0.14.0),
+such as **Creating Resources**, **Prompting for Kubeconfig**, and **Port Forwarding for Services**. Combined with a handful
+of UX/UI improvements for our navigation, better integration of common actions, and various minor bug fixes, we are very
+excited about this release.
+
+## Creating Resources
+![](/img/posts/2020/07/15/apply-yaml.png)
+**Requires Kubernetes 0.16 or Higher** Octant now supports the ablility to create and update resources via the UI. This feature uses the server side apply API of Kubernetes.
+You can apply multiple resources at a time, they are applied in the order that they have been entered. If an error is encountered the apply will halt and not apply any more resources
+and will alert you about the issue.
+
+## Prompting for Kubeconfig
+![](/img/posts/2020/07/15/no-kubeconfig.png)
+
+When starting Octant previously without a valid Kubeconfig, Octant would fail to start and log an error message in to the console. Now you will be prompted to paste a kubeconfig
```

Comment:
Can be re-written as:

Octant would previously failed to start without a valid Kubeconfig and log an error message into the console.